### ISSUE TITLE
Excellent design

### ISSUE THREAD

**Issue by funkyspider** [pierre-muth/absurd-notifier#1]
I would like to modify the 3d printed enclosure to use an old OnePlus smartphone.
I've been trying to model organic 70's shapes like this in fusion360 but my skills aren't quite there yet. I'm a big fan of designs of that era.

Would you mind sharing your fusion360 source so I can learn from your modelling methods and tweak to fit the smartphone.

Thank you.

**Comment by pierre-muth** [comment]
Hi @funkyspider !
Thanks a lot for the nice words !
My bad, the missing fusion360 file was an oversight, I just uploaded it.
I'm not an expert of Fusion360 at all, so please be forgiving about the file organization. I tried many fusion360 features to get the shape, the timeline could be a mess.
Good luck with your design, putting a phone inside is a very nice and clever idea ! 
Have a nice day,
Pierre

**Comment by patfelst** [comment]
I agree the design is great. Thanks I was going to ask for the fusion360 file today too. I found your blog from the Hackaday article.

**Comment by patfelst** [comment]
I hope you don't mind me continuing this issue. I had a quick look at your fusion360 file and when I rolled the history back, the main "70s TV shape" just appears as a purple "form" (see picture below), there's no sketch to define it that I can see. So maybe you designed this main shape in a separate fusion file, or downloaded it from somewhere? I'd also be interested to know how you created the shape - thanks!
<img width="1562" alt="Screen Shot 2022-07-09 at 7 12 04 pm" src="https://user-images.githubusercontent.com/11342773/178100564-bec2b14c-bcad-4cce-8cb8-e21d164339ed.png">

**Comment by pierre-muth** [comment]
Hi @patfelst 
No problem at all to continue this issue. I modeled the shape using the 'form' element, and starting with a Box : 
![image](https://user-images.githubusercontent.com/3978087/178102904-ebe9d545-a2b4-432a-849d-63c98b7a1af6.png)
Then pushing and moving the control segments : 
![image](https://user-images.githubusercontent.com/3978087/178102999-c7d13c36-f439-4a2a-bd01-3b958bd6dc00.png)
Unfortunately, as the shapes, there is no history of the modeling.
You can still edit the form by double-click on the purple form : 
![image](https://user-images.githubusercontent.com/3978087/178103117-887cf976-a6c6-4290-b1fd-2c4db7fa9bdf.png)

Hope that help

**Comment by patfelst** [comment]
Excellent thank you. I've never used forms before so I learned something new.
